### DISCUSSION TITLE
Latest Github Desktop makes Repo Unaccessable.

### DISCUSSION THREAD

**Discussion by Leavoh** [community/community#14962] [Projects and Issues]
I just updated to the latest version, and I CAN NOT press the "Trust Repository.." button, well I can obviously press it, but nothing happens. I restarted the program, my PC, everything, the button does not do anything.

![image](https://user-images.githubusercontent.com/22341463/163440184-c2d48aec-6ef3-4652-b001-ca18fab9a03e.png)

**Comment by jonfairbanks** [comment]
Looks like this is being tracked here: https://github.com/desktop/desktop/issues/14336
